photoeditor/src/main/java/ja/burhanrashid52/photoeditor/Graphic.java

```diff
@@ -0,0 +1,105 @@
+package ja.burhanrashid52.photoeditor;
+
+import android.content.Context;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.ImageView;
+
+/**
+ * Created by Burhanuddin Rashid on 14/05/21.
+ *
+ * @author <https://github.com/burhanrashid52>
+ */
+abstract class Graphic {
```

Comment:
Complement: This is a good name for this class.

Question: Is this meant to eventually contain the brush logic as well, or just emoji/text/sticker?

Reply:
The brush view is created only once and stays on top of the Graphic stack. That cause a few issues #268 . So for now it's for emoji/text/sticker.